x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -572,7 +572,7 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
       defaultMessage: 'Inference',
     }),
     typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.inference', {
-      defaultMessage: 'Uses a trained model to infer against incoming data.',
+      defaultMessage: 'Uses an inference endpoint or a trained model to infer against incoming data.',
```

Comment:
Sorry yes I thought that was the change. I can be an inference id, trained model deployment id or an trained model id. If that is too much text just state it can be an inference id or trained model deployment id. Technically it can be the model id but we want people to use the deployment id really